components/transfer/TransferTokenTabs.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="is-flex mb-5 token-price-container">
+    <div
+      v-for="tab in tabs"
+      :key="tab.value"
+      class="token-price py-2 px-4 is-flex is-align-items-center"
+      :class="{
+        'token-price__active': value === tab.value,
+      }"
+      @click="() => handleTabClick(tab.value)">
+      <img class="mr-2 is-20x20" :src="tab.icon" alt="token" />
```

Comment:
```suggestion
      <img class="mr-2 image is-24x24" :src="tab.icon" alt="token" />
```
? or otherwise size won't work